src/django_components/types.py

```diff
@@ -19,6 +19,9 @@ def __getitem__(self, params: Any) -> "Annotated":  # type: ignore
                 params = (params,)
             return Annotated(self.type_, *params, **self.metadata[1])  # type: ignore
 
+        def __class_getitem__(self, *params: Any) -> "Annotated":  # type: ignore
+            return Annotated(*params)  # type: ignore
```

Comment:
Because I'm using the type annotations in test files, in Python 3.8 our shim is used. And there it turned out that the invocations like `Annotated[str, "css"]` were failing, because the class needed to define `__class_getitem__`